docs/src/usage-examples.md

```diff
@@ -32,7 +32,7 @@ const novu = new Novu(process.env.NOVU_API_KEY);
 await novu.trigger('<TRIGGER_NAME>',
   {
     to: {
-      subscriberId: '<UUNIQUE_IDENTIFIER>',
+      subscriberId: '<UNIQUE_IDENTIFIER>',
```

Comment:
?

Reply:
This is just an example placeholder. You can safely change it.